tests/integration_tests/modules/test_ubuntu_drivers.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from tests.integration_tests.clouds import IntegrationCloud
+from tests.integration_tests.util import verify_clean_log
+
+USER_DATA = """\
+#cloud-config
+drivers:
+    nvidia:
+        license-accepted: true
+"""
+
+
+@pytest.mark.user_data(USER_DATA)
+@pytest.mark.oci
+def test_ubuntu_drivers_installed(session_cloud: IntegrationCloud):
+    with session_cloud.launch(
+        launch_kwargs={"instance_type": "VM.GPU3.1"}
```

Comment:
Good integration test, looks reasonable. I can't seem to get it to succeed locally though. I don't think my region has VM.GPU3.1 availability though `availability_domain = "qIZq:PHX-AD-1" ` I'm also seeing errors from Oracle cloud dashboard UI
```
This shape is either not compatible with the selected image, or not available in the current availability domain.
```
I'll try sorting this with you tomorrow.

Reply:
True. I tried to run it with my credentials, but they do not have permission to run this instance shape. I assumed:

* We need a shape containing an Nvidia card: [vmshapes__vm-gpu](https://docs.oracle.com/en-us/iaas/Content/Compute/References/computeshapes.htm#vmshapes__vm-gpu).
* The CI OCI credentials is allowed to instantiate this kind of shape.

If there is some way I can help, please let me know.

Reply:
Turns out is was a couple of pycloudlib bugs for oracle that broke me
- no jammy support: https://github.com/canonical/pycloudlib/pull/200
- inability to choose the right subnet based on availability_domain: https://github.com/canonical/pycloudlib/pull/201


With those issues fixed I could work through your integration test to finalize the review.